### ISSUE TITLE
Possible to use on non-windows systems?

### ISSUE THREAD

**Issue by themightyoarfish** [potree/PotreeDesktop#40]
Seeing this is electron-based, it should be possible to run on any system. Attempting to `npm install` on macos errors because the electron release url is incorrect: https://github.com/electron/electron/releases/download/v7.0.1/electron-v7.0.1-darwin-arm64.zip

which I assume would affect all other systems also?

**Comment by m-schuetz** [comment]
Works on windows. "darwin-arm64" seems to be mac specific, not sure how and where this is generated. Maybe you could try to update the electron version to the latest version in package.json

**Comment by m-schuetz** [comment]
Yeah, only x64 builds of electron 7.0.1 for darwin/mac: https://github.com/electron/electron/releases/tag/v7.0.1

**Comment by themightyoarfish** [comment]
I put in the latest version instead, after which `npm install` succeeds, but `npm run start` opens an empty application. Too bad.

<img width="1608" alt="image" src="https://user-images.githubusercontent.com/11613312/204095363-b6467f55-e030-42a7-99ae-2a42d70f2ead.png">

**Comment by themightyoarfish** [comment]
Not much interesting logging
```
> npm run start

> potree_desktop@1.7.0 start
> electron .

objc[7410]: Class WebSwapCGLLayer is implemented in both /System/Library/Frameworks/WebKit.framework/Versions/A/Frameworks/WebCore.framework/Versions/A/Frameworks/libANGLE-shared.dylib (0x2282ddb50) and /Users/rasmus/Downloads/source_builds/PotreeDesktop/node_modules/electron/dist/Electron.app/Contents/Frameworks/Electron Framework.framework/Versions/A/Libraries/libGLESv2.dylib (0x108ea6978). One of the two will be used. Which one is undefined.
```

**Comment by m-schuetz** [comment]
Any error messages in the electron/chrome dev tools? (ctrl+shift+i on windows, or view->toggle developer tools)

**Comment by raycaster08** [comment]
It doesn't work on Macs, because **PotreeConverter** is built on the windows platform. You can't run **.exe** on a mac. I am trying to perfect the electron build on different platforms.

**Comment by raycaster08** [comment]
https://github.com/potree/PotreeDesktop/issues/31#issuecomment-1441185660
> The main reason it won't work on m1 is that it won't run **PotreeConverter.exe** on windows platform. I added a PotreeConverter compiled for the mac m1 platform to the [repo](https://github.com/yurui01/PotreeDesktop) and it ran successfully
> 
> ```shell
> $ git clone https://github.com/yurui01/PotreeDesktop.git
> $ cd PotreeDesktop
> $ npm install
> $ electron .
> ```
> 
> <img alt="image" width="1437" src="https://user-images.githubusercontent.com/23043361/220817654-ec6c446c-509f-4d22-8bdf-7234dd25b32c.png">
